### PR TITLE
4 packages from Psi-Prod/Mehari at 0.2

### PR DESCRIPTION
**Pull request by ArtichOwO**
This pull-request concerns:
- `mehari.0.2`: A cross-platform library for building Gemini servers
- `mehari-eio.0.2`: Mehari IO implementation using Eio
- `mehari-lwt-unix.0.2`: Mehari IO implementation using Lwt and Unix bindings
- `mehari-mirage.0.2`: Mehari IO implementation for MirageOS

- `mehari-eio-unix.0.1`: Removal

---
* Homepage: https://github.com/Psi-Prod/Mehari
* Source repo: git+https://github.com/Psi-Prod/Mehari.git
* Bug tracker: https://github.com/Psi-Prod/Mehari/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by Tim-ats-d** [comment]
`mdx` tests have been run before release and are expected to fail due to external module dependence.

**Comment by ArtichOwO** [comment]
We removed MDX support for now, to release Mehari 0.2; it'll return for 0.3 though!

**Comment by Tim-ats-d** [comment]
I think this is ready to merge now.

File: packages/mehari-eio/mehari-eio.0.2/opam [hunk 1]
```diff
@@ -1,17 +1,21 @@
 # This file is generated by dune, edit dune-project instead
```
Comment (kit-ty-kate) [inline packages/mehari-eio/mehari-eio.0.2/opam:1]: Why is `mehari-eio-unix.0.1` being remove? Removal of packages are not accepted in opam-repository except for extreme cases. If it still works fine, could you please add the `flags: deprecated` instead if it's a deprecated package?

Comment (ArtichOwO) [reply packages/mehari-eio/mehari-eio.0.2/opam:1]: Oh right sorry, I didn't know that. It is corrected now!

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**